packages/apps/reputation-oracle/server/src/common/guards/signature.auth.ts

```diff
@@ -54,19 +54,19 @@ export class SignatureAuthGuard implements CanActivate {
     );
     if (
       this.authorizedSignerRoles.includes(AuthSignatureRole.JOB_LAUNCHER) &&
```

Comment:
nit: would be better to jjust check for `escrowData` and throw instead of using conditional chaining: this way it is easier to understand if verification fails due to invalid signature or missing data